llvm/utils/git/code-lint-helper.py

```diff
@@ -255,6 +393,13 @@ def run_linter(changed_files: List[str], args: LintArgs) -> tuple[bool, Optional
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--linter",
+        type=str,
+        choices=["clang-tidy", "doc8"],
```

Comment:
For future enhancements: `Flake8` and `PyLint` are great tools for Python linting. But it'll be necessary to tweak configuration.